server/src/main/java/com/lunchtime/security/JwtUtil.java

```diff
@@ -21,7 +21,9 @@
     private int miliseconds = 1000;
     private int seconds = 60;
     private int minutes = 60;
-    private int hours = 10;
+    private int hours = 24;
+    private int days = 24;
+    private long timeLimit = miliseconds * seconds * minutes * hours * days;
```

Comment:
Why you use both `long and int`?
Maybe it will be better to create privat method `getTimeLimit()`?